src/index.js

```diff
@@ -2,6 +2,7 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-use-before-define */
```

Comment:
- [ ] For best practices and clean code, kindly ensure that you fix all your linters errors instead of disabling them

Reply:
Can you please show me possible ways of how to fix circular dependency (not sure) between functions (In my case, it is between renderTask() and editTask()) in a single file without disabling linters because all I can do is disabling the linters.